### DISCUSSION TITLE
Is there a way to review access/ permissions on all repositories without visiting each one?

### DISCUSSION THREAD

**Discussion by rvallenduuk** [community/community#23739] [New to GitHub]
<p>I’d like to review who has access where across my organisation. We have 100+ repos so visiting each one is not an option.<br>
Is there a way to get a report or export of all permissions on all repos?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>This isn’t really possible in the web UI, I’m afraid. Even in GitHub Enterprise, exporting a report is done repo by repo:</p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/viewing-people-with-access-to-your-repository" class="onebox" target="_blank">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/viewing-people-with-access-to-your-repository</a></p>
<p>You could potentially use the API to speed things up.</p>
<p>There is an endpoint to list organization repos:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <a href="https://developer.github.com/v3/repos/#list-organization-repositories" target="_blank">GitHub Developer</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181079156-e6529ec1-e22b-4a4e-a5ff-762614a96d31.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_10x10.png">

<h3><a href="https://developer.github.com/v3/repos/#list-organization-repositories" target="_blank">Repositories</a></h3>

<p>Get started with one of our guides, or jump straight into the API documentation.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>An endpoint to list which teams have access to the repo, and at what level:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <a href="https://developer.github.com/v3/repos/#list-repository-teams" target="_blank">GitHub Developer</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181079157-6530c90b-b043-4f2e-9a99-94746cf76d30.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_10x10.png">

<h3><a href="https://developer.github.com/v3/repos/#list-repository-teams" target="_blank">Repositories</a></h3>

<p>Get started with one of our guides, or jump straight into the API documentation.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>And an endpoint to list repository collaborators, which would include outside collaborators and org members given access outside of a team:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <a href="https://developer.github.com/v3/repos/collaborators/" target="_blank">GitHub Developer</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181079158-7e3ac5c3-6ce1-42d0-92ab-5d9ec029dbd7.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/26361d706a8a255479f0cb328577f99ad8731275_2_10x10.png">

<h3><a href="https://developer.github.com/v3/repos/collaborators/" target="_blank">Collaborators</a></h3>

<p>Get started with one of our guides, or jump straight into the API documentation.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
